operator/main.go

```diff
@@ -256,7 +264,7 @@ func runOperator(cmd *cobra.Command) {
 		}
 
 		if option.Config.SyncK8sNodes {
-			if err := runNodeWatcher(); err != nil {
+			if err := runNodeWatcher(nodeManager); err != nil {
```

Comment:
Does this function have any effect if `nodeManager` wasn't initialised, i.e. neither of `option.Config.IPAM` cases were applied?

Reply:
Yes, the `runNodeWatcher()` and dependent code will check if `nodeManager` is non-nil and only notify the node manager about modifications if so.

Reply:
Ok, so does it need to be a separate thing, or it could run as one of the tasks inside `nodeManager`?

Reply:
The Kubernetes node watcher is also used for other things besides the IPAM hence it is separate right now.